data/xml/2011.tal.xml

```diff
@@ -0,0 +1,260 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2011.tal">
+  <volume id="1" ingest-date="2023-02-10">
+    <meta>
+      <booktitle>Traitement Automatique des Langues 2011 Volume 52 Numéro 1</booktitle>
+      <editor><first>Éric Villemont</first><last>de La Clergerie</last></editor>
```

Comment:
Should be 
<editor><first>Éric</first><last>Villemonte de La Clergerie</last></editor>